crates/rspack_plugin_devtool/src/lib.rs

```diff
@@ -407,10 +407,7 @@ impl Plugin for SourceMapDevToolPlugin {
                   .chunk(chunk)
                   .filename(&filename)
                   .content_hash_optional(
-                    chunk
-                      .content_hash
-                      .get(source_type)
-                      .map(|i| i.rendered(compilation.options.output.hash_digest_length)),
+                    chunk.content_hash.get(source_type).map(|i| i.rendered(32)),
```

Comment:
Arbitrary picked `32` is not correct. It should be with respect to the algorithm.

Reply:
Yes. You are right.